Makefile

```
@@ -229,6 +236,48 @@ else
+		fi; \
+	else \
+		echo "✅ Docker authentication is working"; \
+	fi
+	
+	# Build directly in the FIPS container without external script
+	# Create local cache directory for CI/ACT compatibility
+	mkdir -p $(PWD)/.build-cache
+	docker run --rm \
+		--entrypoint="" \
+		--user root \
+		-v $(PWD):/plugin \
```

checkmake (0.3.2)

[warning] Target body for "-v $(PWD)" exceeds allowed length of 5 lines (12).

(maxbodylength)